### PR TITLE
Add multistream settings tab

### PR DESCRIPTION
**Pull request by michelinewu**

### CODE REVIEW

File: app/components-react/windows/settings/Multistreaming.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  const { UserService, MagicLinkService } = Services;
+
+  async function upgradeToPrime() {
+    const link = await MagicLinkService.getDashboardMagicLink('prime-marketing', 'slobs-ui-themes');
+    remote.shell.openExternal(link);
+  }
+
+  const shouldShowPrime = UserService.views.isLoggedIn && !UserService.views.isPrime;
+
+  // @@@ TODO: Add correct copy
```
Comment (andycreeth) [inline app/components-react/windows/settings/Multistreaming.tsx:18]: Are all the TODO's in this file still valid?  And are any of them blockers to merging this?

File: app/components-react/windows/settings/Multistreaming.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+export function MultistreamingSettings() {
+  const { UserService, MagicLinkService } = Services;
+
+  async function upgradeToPrime() {
+    const link = await MagicLinkService.getDashboardMagicLink('prime-marketing', 'slobs-ui-themes');
+    remote.shell.openExternal(link);
+  }
+
+  const shouldShowPrime = UserService.views.isLoggedIn && !UserService.views.isPrime;
```
Comment (andycreeth) [inline app/components-react/windows/settings/Multistreaming.tsx:16]: This should be accessed with `useVuex` to make sure it's reactive.  `isLoggedIn` and `isPrime` are things that are unlikely to change while this window is open, but still technically can, so best to hook it up to be reactive.

**Review by andycreeth — CHANGES_REQUESTED** [review]

File: app/components-react/windows/settings/Multistreaming.tsx (hunk 1 above)
Comment (michelinewu) [reply app/components-react/windows/settings/Multistreaming.tsx:18]: Ah, I forgot to delete them - I'll cleanup

File: app/components-react/windows/settings/Multistreaming.tsx (hunk 2 above)
Comment (michelinewu) [reply app/components-react/windows/settings/Multistreaming.tsx:16]: Sounds good!

**Review by andycreeth — APPROVED** [review]

**Pull request merged**